Bottom: d106c62f30a4e950908a2056e41513d5d93eedce
Top:    3f6c457a7ad1117e6bc5b55a737bd1d17fa024c9
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-19 11:33:43 +0800

Refresh of autofs-5.0.7-misc-man-page-fixes.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 0242c11..503a21e 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -51,6 +51,7 @@
 - probe each nfs version in turn for singleton mounts.
 - add changlog entry for coverity fixes.
 - fix probe each nfs version in turn for singleton mounts.
+- misc man page fixes.
 
 25/07/2012 autofs-5.0.7
 =======================
